tests/unit/test_rocket.py

```diff
@@ -496,3 +496,26 @@ def test_get_inertia_tensor_derivative_at_time(calisto):
     assert pytest.approx(0, atol) == inertia_tensor.y[2]
     assert pytest.approx(0, atol) == inertia_tensor.z[0]
     assert pytest.approx(0, atol) == inertia_tensor.z[1]
+
+
+def test_add_cp_eccentricity(calisto):
+    """Test add_cp_eccentricity method of the Rocket class."""
+    calisto.add_cp_eccentricity(0.1, 0.1)
+    assert calisto.cp_eccentricity_x == 0.1
+    assert calisto.cp_eccentricity_y == 0.1
```

Comment:
This test already exists here? https://github.com/RocketPy-Team/RocketPy/blob/54c69a323714b22404c9033f019892f4b0dcdc59/tests/unit/test_rocket.py#L345

Reply:
[36622ad](https://github.com/RocketPy-Team/RocketPy/pull/617/commits/36622adced8e89e9adbf59eec19ec9fb3a90d2e2)